src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -58,6 +61,33 @@ public static TranscriptionResponse mapToTranscriptionResponse(TranscriptionEnti
             transcriptionResponse.setCourthouse(courtCase.getCourthouse().getCourthouseName());
             transcriptionResponse.setDefendants(courtCase.getDefendantStringList());
             transcriptionResponse.setJudges(courtCase.getJudgeStringList());
+            transcriptionResponse.setFrom(transcriptionEntity.getCreatedBy().getUsername());
+
+            if (transcriptionEntity.getTranscriptionStatus() != null) {
+                transcriptionResponse.setStatus(transcriptionEntity.getTranscriptionStatus().getDisplayName());
+            }
+
+            transcriptionResponse.setReceived(transcriptionEntity.getCreatedDateTime());
+
+            // add comments mapped to the workflow that is also mapped to the transaction
+            List<String> comments = transcriptionEntity.getTranscriptionWorkflowEntities().stream()
+                .map(twe -> twe.getTranscriptionCommentEntities().stream()
+                    .map(TranscriptionCommentEntity::getComment))
+                .flatMap(list -> list.toList().stream()).collect(Collectors.toList());
+
+            // if we have at least one workflow then take the first one and set the workflow from and
+            // the date the transcription was received
+            Optional<TranscriptionWorkflowEntity> transcriptionWorkflow
```

Comment:
Think we should really look for the "Requested" transcription_workflow then use that for both the comments and workflow_actor and workflow_ts